source/sass/blocks/form-info.scss

```
@@ -0,0 +1,54 @@
+.form-info__label {
+  display: block;
+  margin: 17px 0;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 18px;
+  box-sizing: border-box;
+}
+
+.form-info__input {
+  display: block;
+  padding: 16px 18px 18px 16px;
+  font-size: 19px;
+  line-height: 18px;
+  color: $basic-dark;
+  border: none;
+  border-bottom: 2px solid $basic-black;
+  box-sizing: border-box;
+}
+
+.form-info__input:hover,
+.form-info__input:active {
+  border-bottom: 2px solid $special-extra-blue;
+}
+
+.form-info__input::-webkit-input-placeholder,
+.form-info__textarea::-webkit-input-placeholder {
+  font-family: inherit;
+  font-size: 17px;
+  line-height: 18px;
+}
+
+.form-info__input:focus::-webkit-input-placeholder,
+.form-info__textarea:focus::-webkit-input-placeholder {
+  opacity: 0;
+  transition: opacity 0.3s ease;
+}
+
+.form-info__textarea {
+  min-height: 142px;
+  margin-top: 22.5px;
+  padding: 16px 18px 18px 16px;
+  font-family: inherit;
+  font-size: 19px;
+  line-height: 18px;
+  color: $basic-dark;
+  border: 2px solid $basic-dark;
+  box-sizing: border-box;
+}
```

Для многострочных полей ввода лучше сразу выключать возможность горизонтального ресайза, чтобы пользователь лэйаут не ломал.
```suggestion
.form-info__textarea {
  min-height: 142px;
  margin-top: 22.5px;
  padding: 16px 18px 18px 16px;
  font-family: inherit;
  font-size: 19px;
  line-height: 18px;
  color: $basic-dark;
  border: 2px solid $basic-dark;
  box-sizing: border-box;
  resize: vertical;
}
```